apps/api/src/auth/service/auth.service.ts

```diff
@@ -29,19 +29,12 @@ export class AuthService {
     this.logger = new Logger(AuthService.name)
   }
 
-  async sendOtp(email: string): Promise<void> {
-    if (!email || !email.includes('@')) {
-      this.logger.error(`Invalid email address: ${email}`)
-      throw new HttpException(
-        'Please enter a valid email address',
-        HttpStatus.BAD_REQUEST
-      )
-    }
-
+  private async createUserIfNotExists(email: string) {
```

Comment:
Since we will get stuff like `name`, `profilePicture` aswell. Then, pass them down to the `create` function.

Reply:
done. ✅